CMakeLists.txt

```diff
@@ -224,19 +224,23 @@ if(USE_CUDA)
     # reference for mapping of CUDA toolkit component versions to supported architectures ("compute capabilities"):
     # https://en.wikipedia.org/wiki/CUDA#GPUs_supported
     set(CUDA_ARCHS "60" "61" "62" "70" "75")
-    if(CUDA_VERSION VERSION_GREATER_EQUAL "110")
```

Comment:
"110" means exactly 110 version during comparison, `VERSION_GREATER_EQUAL` doen't know whether and where we want to put a `.`: `11.0` or maybe `1.10`.